src/Extractor/SnowsqlExportAdapter.php

```diff
@@ -113,6 +113,12 @@ private function runDownloadCommand(ExportConfig $exportConfig, string $csvFileP
 
         // Check result
         if (!$process->isSuccessful()) {
+            if (str_contains(
```

Comment:
Nechápu

Reply:
no když něco exportuješ přes snowsql a je tam 0výsledků tak nově snowsql vyhodí chybu jakože soubor neexistuje

Reply:
a my neočekáváme dál Exception ale prázdný výsledek

Reply:
takže tu exceptionu odchytím a ten prázdný process prostě vrátím

Reply:
a tenhle problém je jen tady, protože writery u snowsql nepoužívají export ale import

Reply:
Hoď to tam prosím do komentáře, tohle je něco, co podle mě na první dobrou nikomu nedojde.

Reply:
máš to tam